platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java

```diff
@@ -267,6 +283,31 @@ public void createNextExperiment() {
         }
     }
 
+    private void createRabbitMQ(ExperimentConfiguration config) throws Exception {
+        String rabbitMQAddress = EnvVariables.getString(RABBIT_MQ_EXPERIMENTS_HOST_NAME_KEY, (String)null);
```

Comment:
Please do not forget to update the documentation accordingly (i.e., that there are different ways to handle the communication of an experiment: a) either the broker is created while starting the platform and the platform uses it or b) the platform creates a new broker for each experiment).